### PR TITLE
feat: added cta icons

### PR DESCRIPTION
**Pull request by GutierrezGo**
References https://autoricardo.atlassian.net/browse/DM-920

## Before

No cta icons

## After

Cta icons

## How to test

Please check the icons story in the theme category

### CODE REVIEW

File: src/components/icons/CallCtaIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { createIcon } from '@chakra-ui/react';
+
+export const CallCtaIcon = createIcon({
+  displayName: 'CallCta',
+  viewBox: '0 0 24 24',
+  path: (
+    <path
+      d="m13.4998 20c-.7935.0019-1.5794-.1535-2.3123-.4573-.733-.3038-1.39838-.75-1.95774-1.3127l-6.46-6.46c-.56539-.5581-1.01432-1.223-1.32072-1.95598-.3064-.733-.464177-1.51956-.464177-2.31402s.157777-1.58102.464177-2.31402.75533-1.39786 1.32072-1.95598l3.23-3.23 6.00004 6-3.00004 3 3.00004 3 3-3 6 6-3.23 3.23c-.5594.5627-1.2248 1.0089-1.9578 1.3127-.7329.3038-1.5188.4592-2.3122.4573zm-7.50004-17.17-1.82 1.82c-.37794.3722-.67808.81585-.88294 1.30513-.20487.48928-.31037 1.01443-.31037 1.54487s.1055 1.05559.31037 1.54487c.20486.48928.505.93293.88294 1.30513l6.47004 6.47c.3733.3755.8174.6732 1.3066.8758.4893.2025 1.0139.3059 1.5434.3042.5311.0031 1.0576-.0997 1.5487-.3023.491-.2026.9368-.501 1.3113-.8777l1.81-1.82-3.17-3.17-3 3-5.83004-5.83 3-3z"
+      fill="#333"
```
Comment (talamcol) [inline src/components/icons/CallCtaIcon.tsx:10]: should we use currentColor here?

File: src/components/icons/index.ts [hunk 2]
```diff
@@ -10,8 +10,10 @@ export { ChevronLeftSmallIcon } from './ChevronLeftSmallIcon';
 export { ChevronRightSmallIcon } from './ChevronRightSmallIcon';
 export { ChevronRightTinyIcon } from './ChevronRightTinyIcon';
 export { ErrorIcon } from './ErrorIcon';
+export { CallCtaIcon } from './CallCtaIcon';
```
Comment (talamcol) [inline src/components/icons/index.ts:13]: I would suggest to remove the `Cta`. This says what it should be used for and not what it represents. I could use this icon for other things.
`CallIcon` and `MessageIcon`

**Review by talamcol — APPROVED** [review]
lovely

File: src/components/icons/CallCtaIcon.tsx (hunk 1 above)
Comment (GutierrezGo) [reply src/components/icons/CallCtaIcon.tsx]: Yes, I forgot to change it. Thanks 🙌🏼

**Pull request merged**